scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -622,6 +647,28 @@ class FormatWriter(formatOps: FormatOps) {
     }
   }
 
+  private def prepareAlignmentInfo(
+      block: Vector[Array[FormatLocation]],
+      separatorLengthGaps: Array[Int],
+      column: Int
+  ): Vector[AlignmentUnit] =
+    block.zipWithIndex.map {
+      case (line, i) =>
+        val location = line(column)
+        val previousWidth =
+          if (column == 0) 0 else line(column - 1).shift
+        val key = location.shift - previousWidth + separatorLengthGaps(i)
+        val separatorLength =
+          if (location.formatToken.right.is[Token.Comment]) 0
+          else location.formatToken.right.text.length
```

Comment:
is `text` better than (or different from) `syntax` (which is what i've seen used commonly)?

Reply:
`text` comes from [Position](https://github.com/scalameta/scalameta/blob/master/scalameta/inputs/shared/src/main/scala/scala/meta/inputs/Position.scala#L20)

`syntax` is a `text` with dialect check. `syntax` would be more correct to use here. Replaced.